AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -289,17 +292,17 @@ + (sqlite3 *)openDatabaseAtFileURL:(NSURL *)fileURL withResult:(int *)result {
 
 + (long)getPageSizeInOpenedDatabase:(void *)db {
   NSArray<NSArray *> *rows = [MSDBStorage executeSelectionQuery:@"PRAGMA page_size;" inOpenedDatabase:db];
-  return [(NSNumber *)rows[0][0] longValue];
+  return rows.count > 0 && rows[0].count > 0 ? [(NSNumber *)rows[0][0] longValue] : 0;
```

Comment:
I think that this is something that we can move to a helper method so we don't have the same line of code in multiple places.